### DISCUSSION TITLE
Getting Caps/num/Scroll lock information from host?

### DISCUSSION THREAD

**Discussion by experimentech** [NicoHood/HID#304] [Ideas]
Hi there. I've been digging through the sources and documents and haven't found a way to get the status of the locks from the host so I can set LEDs to mirror them. I've written an adapter to use a WYSE terminal keyboard as an NKRO keyboard. In fact that's what I'm using to type this. This library has been invaluable for doing it. However I have no way of getting the status from the host.

I found [this](https://forum.arduino.cc/t/leonardo-keyboard-leds-emulation/169582/14) link related to another library so it seems to be possible, but doesn't seem to have an analog in this library.

So, have I missed it, or is it not there? If it's the latter, it's about the only thing missing from making this pretty much featuer complete. TIA.

**Comment by NicoHood** [comment]
https://github.com/NicoHood/HID/blob/master/examples/Keyboard/KeyboardLed/KeyboardLed.ino#L32
